protocol.md

```diff
@@ -1209,6 +1214,18 @@ _Response_:
 * result: [`Location`](#location) | [`Location`](#location)[]
 * error: code and message set in case an exception happens during the definition request.
 
+#### Get Symbol Request
+
+The get symbol request is sent from the client to the server to resolve the definition location and symbol information for a given text document position.
+
+_Request_
+* method: 'textDocument/symbol'
+* params: [`TextDocumentPositionParams`](#textdocumentpositionparams)
+
+_Response_:
+* result: `SymbolInformation` | `SymbolInformation`[]
+* error: code and message set in case an exception happens during the definition request.
```

Comment:
Actually, I'm even a little more confused. This change looks completely unrelated to the PR description + `extension-glob-content.md`, hmm